pkgs/tools/misc/bunnyfetch/default.nix

```diff
@@ -1,28 +1,31 @@
 { buildGoModule, fetchFromGitHub, lib }:
 
-buildGoModule rec {
-  pname = "bunnyfetch";
-  version = "unstable-2021-05-24";
+let buildGoModule116 = buildGoModule.override {
+  go = buildPackages.go_1_16;
+};
+in
+  buildGoModule rec {
+    pname = "bunnyfetch";
+    version = "24370338b936bae1ebdefea73e8372ac0b4d2858";
 
-  src = fetchFromGitHub {
-    owner = "Mewyuna";
-    repo = pname;
-    rev = "7bcc45fb590b37f410e60af893e679eb0729ecb1";
-    sha256 = "1lgqrwmxdxd1d99rr0akydfwcsbcmz75fkbq9zrl842rksnp5q3r";
-  };
+    src = fetchFromGitHub {
+      owner = "Rosettea";
+      repo = "bunnyfetch";
+      rev = "${version}";
```

Comment:
```suggestion
      rev = "${version}";
      rev = version;
```

Reply:
Also, you should probably undo the noise caused by whatever formatter was used for the file, the indents were fine as they were.

Reply:
Applied the version fix.

The indents were to match the new block introduced form the `let ... in` syntax. I removed this with the latest commit